tofu/geom/_core.py

```diff
@@ -4576,9 +4577,10 @@ def calc_signal(self, ff, t=None, ani=None, fkwdargs={}, Brightness=True,
         # Exclude Rays not seeing the plasma
         s = _GG.LOS_calc_signal(ff, Ds, us, res, DL,
                                 dmethod=resMode, method=method,
-                                t=t, Ani=ani, fkwdargs=fkwdargs, Test=True)
+                                t=t, ani=ani, fkwdargs=fkwdargs, Test=True)
         if t is None or len(t)==1:
             sig[indok] = s
+            sig = sig.reshape((1,len(sig)))
```

Comment:
This might not be needed, I added it wanting to make it the array 2D when creating the 2D camera in post treatment, however when the camera is created, it "squeezes" the array, and flattens it again